snow/consensus/snowball/binary_snowball_test.go

```diff
@@ -19,25 +19,26 @@ func TestBinarySnowball(t *testing.T) {
 	beta := 2
 	terminationConditions := newSingleTerminationCondition(alphaConfidence, beta)
 
-	sb := newBinarySnowball(alphaPreference, terminationConditions, red)
-	require.Equal(red, sb.Preference())
-	require.False(sb.Finalized())
+	sf := newBinarySnowflake(alphaPreference, terminationConditions, red)
```

Comment:
Why is this test modified? It seems to be a duplicate of `TestBinarySnowflake` now.